src/ui/pages/GenerateSeedPhrase/GenerateSeedPhrase.tsx

```diff
@@ -100,28 +76,13 @@ const GenerateSeedPhrase = () => {
     );
   };
 
-  const HandlePrivacy = () => {
-    return (
-      <u
-        data-testid="privacy-policy-modal-handler"
-        onClick={() => setPrivacyModalIsOpen(true)}
-      >
-        {i18n.t("generateseedphrase.termsandconditions.privacy")}
```

Comment:
Confirmed by Robert that we no longer have a privacy policy here?

If so, we can remove "generateseedphrase.termsandconditions.privacy" from i18n